doc/covid-response.Rmd

```diff
@@ -0,0 +1,106 @@
+---
+title: "Covid19 Response Ratio Comparison between Canada and USA"
+author: "Fatime Selimi, Neel Phaterpekar, Nicholas Wu, Tanmay Sharma"
+date: "2020/11/27"
+bibliography: covid19.bib
+output: 
+  html_document:
+    toc: true
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+library(knitr)
+library(tidyverse)
+```
+
+# Aim
+
+This project explores if there is a quantifiable difference in the Covid-19 responses of Canada and USA as measured through analyzing the number of daily new cases and daily new tests being conducted in both the countries between March and October 2020.
+
+# Introduction
+
+Coronavirus disease 2019 (COVID-19) is a contagious disease caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) [@who]. North-America has been severely impacted by COVID-19; a quarter of all global deaths as of November 27th, 2020 have occurred on the continent. Observing the pandemic's impact at a country level shows a stark contrast in the number of cases and deaths between Canada as USA. As of November 27th, 2020, there have been 241,906 deaths in the US and 11,799 deaths in Canada as per the countries' official sources [@cdc] [@govCA]. Even if we account for some factors such as population densities, number of international airports, migration of people, climatic factors, etc. to understand the varied impact of COVID-19 on Canada and USA it still seems worthwhile to explore how the response measures of both the countries have shaped the pandemic's trajectory since it first broke out on the continent in January 2020 [@first-case]. Moreover, the widespread coverage and analysis of the COVID-19 in global media as well as different studies in the academic community have led to a popular perception that Canada has been able to better respond to the pandemic than USA. For instance, a study by Jeffrey Lazarus et al.[@covid-score] devised a `covid-score` metric to quantify the government response measures for different countries and ranked Canada (score = 61.00) much better than the USA (covid-score = 50.57). This project attempts a preliminary analysis of the COVID-19 data regarding daily cases and tests being conducted in Canada and USA to understand the response of the these two neighbors to one of the deadliest pandemics that the world has seen in decades.
```

Comment:
Can we remove `et al.` part in "Jeffrey Lazarus et al.[@covid-score]" since it becomes redundant.  The reference is showing it exactly the same as what we have in the text, .i.e. (Lazarus et al. 2020). 

See the screenshot below:
![Screen Shot 2020-11-28 at 10 34 05 AM](https://user-images.githubusercontent.com/24602483/100523423-5441fa00-3165-11eb-9214-bbcba20525d0.png)

Reply:
Acknowledged and corrected.